smart_proxy.go

```diff
@@ -100,6 +106,36 @@ func startServer() int {
 	return ExitStatusOK
 }
 
+// updateGroupInfo
```

Comment:
just to clarify (repeating what you wrote in IRC :):
please update comment a bit:
```
the updateGroupInfo **function** is run in a goroutine. It is always, in loop, waiting for one of 2 events: a ticket or a channel.
- if the ticket comes first, it tries to update the groups configuration from the content-service and, if success, it updates the current configuration
- if the channel comes first, then it send the latest retrieved configuration
```

Reply:
I just updated the "doc strings" for `updateGroupInfo` and `getGroups`, where the channel is used

Reply:
Are you ok with the new comments in the last version?

Reply:
yes, TY